cmd/loop/liquidity.go

```diff
@@ -399,3 +399,24 @@ func suggestSwap(ctx *cli.Context) error {
 
 	return nil
 }
+
+var forceTickCmd = cli.Command{
```

Comment:
Do we even need a CLI for this? I assume we'll want to trigger this over RPC in the itests only. If people shouldn't use it, why give them a command?

Reply:
Included it because I used it a bit for testing, but agreed we don't need it. Removed.